src/containers/coming-soon/coming-soon.scss

```diff
@@ -99,7 +144,53 @@
   }
 
   footer {
-    padding: 100px 10px 10px;
-    text-align: center;
+    flex: 0 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 20px 66px;
+    margin-top: 20px;
+    .email-link {
+      font-family: Inter;
+      font-size: 24px;
+      line-height: 116.4%;
+      text-transform: uppercase;
+      color: $primary;
+    }
+    .medium-link {
+      margin-right: 30px;
+    }
+  }
+}
+
+@media (max-width: 768px) {
```

Comment:
I recommend you to create constants-file with width break-points of screens. It will be more convenient for you.